### PR TITLE
Fix icons

### PR DESCRIPTION
**Pull request by ralphtheninja**
Solves https://github.com/deltachat/deltachat-desktop/issues/59 and solves point 2 in https://github.com/deltachat/deltachat-desktop/issues/58

screenshot from 2018-09-25 00-29-14
screenshot from 2018-09-25 00-29-44

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -84,9 +84,6 @@
       ]
     },
     "appId": "com.deltachat.desktop",
-    "directories": {
-      "buildResources": "src"
-    },
     "linux": {
       "target": [
         "AppImage",
```
Comment (ralphtheninja) [inline package.json]: wth is this red coloring?

File: package.json [hunk 2]
```diff
@@ -84,9 +84,6 @@
       ]
     },
     "appId": "com.deltachat.desktop",
-    "directories": {
-      "buildResources": "src"
-    },
```
Comment (ralphtheninja) [inline package.json]: This hack is not needed anymore since we build to `es5/` folder.

File: package.json (hunk 1 above)
Comment (karissa) [reply package.json]: missing an empty double quote?

File: package.json (hunk 2 above)
Comment (karissa) [reply package.json]: woot

**Pull request merged**